docusaurus.config.js

```diff
@@ -47,21 +47,26 @@ const config = {
         //disableSwitch: true
       },
       navbar: {
-        title: 'ColourBox',
-
         logo: {
           alt: 'ColourBox Logo',
-          src: 'img/logo.svg',
+          src: 'img/colourbox_logo_label.png',
         },
 
         items: [
+          { to: '/blog', label: 'RESOURCES', position: 'right' },
           {
             type: 'doc',
             docId: 'intro',
-            position: 'left',
-            label: 'Getting started',
+            position: 'right',
+            label: 'FAQs',
+          },
+          { to: '/blog', label: 'CONTACT US', position: 'right' },
+          {
+            to: 'https://rinkeby.staging.colourbox.io/',
+            label: 'GET STARTED',
```

Comment:
don't put link to test site, Remove this for now, we will add this when the Live is available